packages/kbn-generate-csv/src/generate_csv.ts

```diff
@@ -75,6 +79,7 @@ export class CsvGenerator {
         {
           requestTimeout: duration,
           maxRetries: 0,
+          maxConcurrentShardRequests,
```

Comment:
This also needs to be added to the `transport` field of the `data.search()` call in the `doSearch()` method. Doing that will throw a type compilation error, since the field isn't known in the data service. You'll have to reach out to them and make sure the field is recognized and correctly sent in the requests

Reply:
Omg thank you good catch

Reply:
The params for ``
I did some searching around to find how this is handled in other parts of the code, and I think `max_concurrent_shard_requests` must be in the body, not the options. 
```typescript
      const response = await this.clients.es.asCurrentUser.openPointInTime(
        {
          // Note, this throws a TypeScript error, same as "ignore_throttled"
          // Check with the ES clients team to make sure this will work
          max_concurrent_shard_requests: maxConcurrentShardRequests,
          index: indexPatternTitle,
          keep_alive: duration,
          ignore_unavailable: true,
          // @ts-expect-error ignore_throttled is not in the type definition, but it is accepted by es
          ignore_throttled: includeFrozen ? false : undefined, // "true" will cause deprecation warnings logged in ES
        },
        {
          requestTimeout: duration,
          maxRetries: 0,
        }
      );
```